docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,234 @@
+# Preparing the LFRQA Dataset
+
+## Overview
+
+The **LFRQA dataset** was introduced in the paper [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998). It features **1,404 science questions** (along with other categories) that have been human-annotated with answers. This tutorial walks through the process of setting up the dataset for use.
+
+## Step 1: Download the Annotations
+
+First, we need to obtain the annotated dataset from the official repository:
+
+```bash
+# Clone the RAG-QA Arena repository
+!git clone <https://github.com/awslabs/rag-qa-arena>
```

Comment:
Okay so we have a `bash` language tag, so no need for `!`.

If you want `!`, then tag this as `jupyter`

Reply:
I will tag everything as jupyter so that you can just copy the tutorial to a notebook, is that ok?